examples/io/open_save/read_satimg.py

```diff
@@ -0,0 +1,23 @@
+"""
+Parsing metadata
+================
+
+This example demonstrates the instantiation of an image through :class:`~geoutils.SatelliteImage`.
+"""
+
+import geoutils as gu
+
+# %%
+# We print the filename of our raster that, as often with satellite data, holds metadata information.
+filename = gu.examples.get_path("everest_landsat_b4")
+import os
+
+print(os.path.basename(filename))
+
+# %%
+# We open it as a geo-image, unsilencing the attribute retrieval to see the parsed data.
+img = gu.SatelliteImage(gu.examples.get_path("everest_landsat_b4"), silent=False)
+
+# %%
+# We have now retrieved the metadata. For the rest, the :class:`~geoutils.SatelliteImage` is a subclass of :class:`~geoutils.Raster`, and behaves similarly.
```

Comment:
You should explain in which attributes the metadata are stored.

Reply:
It's shown in the print statement, and might change very soon